### ISSUE TITLE
Version bumps for new Stack nightlies?

### ISSUE THREAD

**Issue by theindigamer** [jtdaugherty/brick#194]
The latest brick version on Stackage is [0.38](https://www.stackage.org/nightly-2018-09-28/package/brick-0.38), corresponding to Nightly 2018-09-28. Do you need help with updating version bounds etc. to get it up to date with latest Stackage nightlies including GHC 8.6.1?

**Comment by jtdaugherty** [comment]
I'm not quite sure what you're asking. Are you offering to do that work? If so, what version bounds need updating?

**Comment by theindigamer** [comment]
Yes, I'm offering to help out :smile: . Brick is available in LTS and not in nightlies -- my guess is this happened because one of Brick's transitive dependencies is not available in nightly. That might either be because of (a) an upstream versioning problem and/or (b) because the version bounds in Brick are too restrictive.

I'm not sure which case it is (a) or (b) or both, so I opened this issue to ask, and possibly help out with the investigation. Sorry for not being clear in the first comment :sweat_smile:.

---

Actually, I just looked at [packdeps](http://packdeps.haskellers.com/feed?needle=brick) (forgot about that site earlier) and it looks like your dependencies are up to date, so I'm guessing it is an upstream problem. Sorry for the false alarm!
